src/tests/fableconfig.json

```diff
@@ -13,7 +13,7 @@
             "projFile": ["Main/project.fsx"],
             "ecma": "es2015",
             "module": "commonjs",
-            "coreLib": "../../build/fable-core/umd",
+            "coreLib": "../../build/fable-core",
```

Comment:
Great! I added you as collaborator to the `fable-compiler-netcore` pkg so now you've publish rights :+1: Don't worry about the output of fable-core, I think both versions of fable-compiler can share the same fable-core dependency, so building fable-core in `FableCompilerNetcore` FAKE target is just to pass the tests.

There's already a `PublishCompilerNetcore` target that should work (once you add your npm credentials to your machine, check the npm site for that), it just runs `npm publish` from the build/fable dir (adding `--tag next` if needed). For fable-compiler I do the builds in AppVeyor and there I remove the *.pdb and *.xml files to make the package lighter (check `MakeArtifactLighter` target), you may want to do that as well.

Please tell me if you've any question, and thanks a lot for your help!